tservice/src/seal.rs

```diff
@@ -68,6 +91,24 @@ impl<T: AsRef<[u8]> + core::default::Default> SealedBuilder<T> {
         self
     }
 
+    /// Set the key policy to use for the sealed data
+    ///
+    /// # Arguments
+    /// * `policy` - The key policy to use
+    pub fn key_policy(mut self, policy: KeyPolicy) -> Self {
+        self.policy = policy;
+        self
+    }
+
+    /// Set the attributes to use for the sealed data
+    ///
+    /// # Arguments
+    /// * `attributes` - The attributes to use when sealing
+    pub fn attributes(mut self, attributes: Attributes) -> Self {
```

Comment:
I'm a little hesitant to expose the `Attributes` "raw" like this. For one thing, sealing will fail if `INIT` and `DEBUG` aren't included in the attribute mask. For another, most of the attribute mask is reserved bits, as are most of the xfrm bits (one could imagine implementing this in the `Attributes` struct's methods, having them take a bitflags struct instead or something). Finally, Intel's guide pretty strongly recommends not moving off of the defaults for this, and I was not able to figure out what this field actually _does_ in the key generation process despite diving pretty deep.

Given the above, and the fact that this method is only called with defaults in the mobilecoin repo, I recommend removing this function entirely.

Reply:
Good catch.  Looking further it doesn't appear that `sgx_seal_data_ex` is actually used for the `_ex` part in the mobilecoin repo.  I think the key policy expansion is "ok", but am also fine closing this PR if we think it's not worth doing at this time.